src/applications/vaos/services/appointment/transformers.js

```diff
@@ -115,16 +114,17 @@ function getStatus(appointment, isPast) {
       return APPOINTMENT_STATUS.booked;
     case APPOINTMENT_TYPES.ccRequest:
     case APPOINTMENT_TYPES.request: {
-      if (
-        appointment.status === 'Booked' ||
-        appointment.status === 'Resolved'
-      ) {
+      if (appointment.status === 'Booked') {
         return APPOINTMENT_STATUS.booked;
+      } else if (appointment.status === 'Resolved') {
+        return APPOINTMENT_STATUS.fulfilled;
```

Comment:
I wasn't too sure about handling this 'Resolved' request this status.  Hopefully it doesn't get confused as a resolved appointment, but I guess I can't think of too many scenarios where we'd need to refer to it.

Reply:
Yeah, I may change it back just to be safe. I mainly don't want to have EC requests get marked as "booked," since they're not ever a booked appointment, as far as I know. Ubt I think these status mean something different for EC requests in general and maybe we'll need to have a separate function for setting them to make it less confusing.

Reply:
@narin Changed back